README.md

```diff
@@ -1,2 +1,175 @@
-# ruby-capstone-kanban
-This repository is created to setup kanban board
+https://github.com/Xghayor/ruby-group-capstone.git
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ruby-group-capstone] <a name="about-project"></a>
+
+**ruby-group-capstone** Is a console app that helps keep records of different types of things: **books**, **music albums**, and **games**.The data is stored in JSON files. A database with tables structure analogical to the program's class structure is prepared.
+
+## Video Presentation [https://drive.google.com/file/d/1k07eGHkiUsOLzgWb3zWzl5UdJUATmB3m/view?usp=sharing]
```

Comment:
- [ ] Your video has improved but it still has a major part missing, and that is the interesting piece of code in your project. Kindly record a new one where you won't have to do much editing and trimming to ensure that everything is clear and consistent.
- [ ] [OPTIONAL] As referenced by previous reviewer, for good user experience, kindly format your video link well such that it appears clickable and in appropriate font-size. I suggest taking the link to the second line after your heading "Video Presentation" and add the link in this manner:
```
[Video presentation link](https://drive.google.com/file/d/1k07eGHkiUsOLzgWb3zWzl5UdJUATmB3m/view?usp=sharing)

Reply:
@AbayomiOlaoye Hi, 

Thanks for the review, However could you please check that the last CR made it optional, and the video was okay

Reply:
- I really love your attitude to feedback :clap: :clap: but if you check, I am also making the video link formatting an optional change but the video is a major requirement for this project, and the previous reviewer indicated that it was a required change if you check the screenshot below:

![image](https://github.com/Xghayor/comt-console-app/assets/99715640/1eee0361-97f4-4338-b1ae-c7927dff801b)

Reply:
@AbayomiOlaoye 

I apologize for not having noticed the required option. We will make the necessary updates soon.

Reply:
It's fine, @Xghayor :wave: 
I trust you will. All the best :100: